FlowerinoPlugin/src/org/flowerplatform/flowerino_plugin/command/HeartbeatCommand.java

```diff
@@ -0,0 +1,21 @@
+package org.flowerplatform.flowerino_plugin.command;
+
+import org.flowerplatform.tiny_http_server.FlexResponse;
+import org.flowerplatform.tiny_http_server.IHttpCommand;
+
+/**
+ * Heartbeat command, allowing a client to verify if the Arduino IDE is available or 
+ * not (i.e. started). 
+ * 
+ * @author Andrei Taras
+ */
+public class HeartbeatCommand implements IHttpCommand {
+	
+	public static final String MESSAGE_OK = "Ok";
+	
+	@Override
+	public Object run() {
+		//Don't actually do anything, but return a simple status.
+		return new FlexResponse(FlexResponse.CODE_OK, MESSAGE_OK);

```

Comment:
IHttpCommand: sa il facem generic, avand ca parametru tipul de retur.
Apoi toate metodele noastre ar trebui sa returneze FlexResponse, sau subclase.
Apoi FlexResponse ar trebui sa se numeasca poate mai simplu Response, avand in vedere ca de fapt nu Flex o sa fie cel care apeleaza.